lib/dropsonde.rb

```diff
@@ -1,9 +1,12 @@
+# frozen_string_literal: true
+
 require 'json'
 require 'httpclient'
 require 'puppetdb'
 require 'inifile'
 require 'puppet'
 
+# Dropsonde class documentation
```

Comment:
:eyes:

Reply:
Sorry about that. I forgot to add a description for Dropsonde class. I put that comment just to avoid rubocop failures.
I will add now.